code/game/objects/items/devices/traitordevices.dm

```diff
@@ -262,3 +262,24 @@ effective or pretty fucking useless.
 	else
 		GLOB.active_jammers -= src
 	update_icon()
+
+/obj/item/storage/toolbox/emergency/turret
+	desc = "You feel the need to hit this one with a wrench"
+
+/obj/item/storage/toolbox/emergency/turret/PopulateContents()
+	new /obj/item/screwdriver(src)
+	new /obj/item/wrench(src)
+	new /obj/item/weldingtool(src)
+	new /obj/item/crowbar(src)
+	new /obj/item/analyzer(src)
+	new /obj/item/wirecutters(src)
+
+/obj/item/storage/toolbox/emergency/turret/attackby(obj/item/I, mob/living/user, params)
+	..()
```

Comment:
won't this cause the wrench to be placed inside of the toolbox just before it transforms?

Reply:
In testing I didn't notice it happening but let me check again

Reply:
Ah it does, let me fix that